django/river/migrations/0002_batch_mappings.py

```diff
@@ -10,10 +10,6 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RemoveField(
-            model_name="batch",
-            name="resources",
-        ),
```

Comment:
is that what we really want, editing an existing migration ?

Reply:
this one has never been merged in main, hence never been deployed. so it's fine?

Reply:
yes it's fine!